.github/workflows/deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+      # TODO: Uses for testing this pull request only. Remove before merge!
```

Comment:
Reminder that we'll want to remove this comment and related line of code before merging this in.